common/connection.go

```diff
@@ -566,10 +565,12 @@ func (c *Connection) sendLoop() {
 
 // Close cleanly closes the WebSocket connection.
 // It returns an error if sending the Close control frame fails.
-func (c *Connection) Close(args ...sobek.Value) {
-	code := websocket.CloseGoingAway
+//
+// Optional code to override default websocket.CloseGoingAway (1001).
+func (c *Connection) Close(args ...int) {
```

Comment:
I wonder if we should just remove the `args` argument from `connection.Close`. No calls use the optional argument.

Reply:
👍